src/Runner.Listener/CommandSettings.cs

```diff
@@ -249,6 +250,24 @@ public string GetStartupType()
             return GetArg(Constants.Runner.CommandLine.Args.StartupType);
         }
 
+        public ISet<string> GetLabels(string description)
+        {
+            var labelSet = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+            string labels = GetArgOrPrompt(
+                name: Constants.Runner.CommandLine.Args.Labels,
+                description: description,
+                defaultValue: string.Empty,
```

Comment:
do we need to have both `defaultValue` and `isOptional`? the `defaultValue` suppose to cover the optional scenario.

Reply:
I would like to target scenarios where the default value is `string.Empty` and the value is optional, so that we can display `[Enter to skip]` which speaks out it's optional parameter.

Reply:
cool. 👍